docs/deploy-rbd.md

```diff
@@ -50,6 +50,8 @@ make image-cephcsi
 | `clusterID`                                                                                         | yes                  | String representing a Ceph cluster, must be unique across all Ceph clusters in use for provisioning, cannot be greater than 36 bytes in length, and should remain immutable for the lifetime of the Ceph cluster in use |
 | `pool`                                                                                              | yes                  | Ceph pool into which the RBD image shall be created                                                                                                                                                                     |
 | `dataPool`                                                                                          | no                   | Ceph pool used for the data of the RBD images.                                                                                                                                                                          |
+| `volumeNamePrefix`                                                                                  | no                   | Prefix to use for naming RBD volume images (defaults to `csi-vol-`).                                                                                                                                                    |
```

Comment:
this applies for cephfs also right not only for rbd? same question for snapshot also